### PR TITLE
add espresso to input fields

### PR DESCRIPTION
**Pull request by ZzzzHui**

### CODE REVIEW

File: internal/reader/reader.resolvers.go [hunk 1]
```diff
@@ -138,3 +139,16 @@ type noticeResolver struct{ *Resolver }
...
 type voucherResolver struct{ *Resolver }
+
+// !!! WARNING !!!
+// The code below was going to be deleted when updating resolvers. It has been copied here so you have
+// one last chance to move it out of harms way if you want. There are two reasons this happens:
+//   - When renaming or deleting a resolver the old code will be put in here. You can safely delete
+//     it when you're done.
+//   - You have helper methods in this file. Move them out to keep these resolver files clean.
+func (r *inputResolver) EspressoTimestamp(ctx context.Context, obj *model.Input) (string, error) {
+	panic(fmt.Errorf("not implemented: EspressoTimestamp - espressoTimestamp"))
```
Comment (ZzzzHui) [inline internal/reader/reader.resolvers.go:150]: where should this be implemented

File: internal/convenience/repository/input_repository.go [hunk 2]
```diff
@@ -390,6 +413,7 @@ func parseInput(res *sqlx.Rows) (*model.AdvanceInput, error) {
 	input.Payload = common.Hex2Bytes(payload)
 	input.MsgSender = common.HexToAddress(msgSender)
 	input.BlockTimestamp = time.UnixMilli(blockTimestamp)
+	input.EspressoBlockTimestamp = time.UnixMilli(espressoBlockTimestamp)
 	input.PrevRandao = prevRandao
```
Comment (ZzzzHui) [inline internal/convenience/repository/input_repository.go:417]: curious whether we could remove the var `prevRandao` and directly use `&input.PrevRandao` when `res.Scan`

Comment (fabiooshiro) [reply internal/convenience/repository/input_repository.go:417]: It's just to keep the style. I see no problem.

File: internal/reader/reader.resolvers.go (hunk 1 above)
Comment (ZzzzHui) [reply internal/reader/reader.resolvers.go:150]: any insight on this one? @fabiooshiro

**Comment by ZzzzHui** [comment]
@fabiooshiro and Bruno,
Does the CI fail because the `graphile_fetcher_test` function here doesn't have `espressoTimestamp` fields?
https://github.com/Calindra/nonodo/blob/0c4396c5cd45e456be1d64905c9f6e56006958d0/internal/convenience/synchronizer/graphile_fetcher_test.go#L150

**Comment by ZzzzHui** [comment]
`espressoBlockNumber` for L1 inputs is now 0 rather than a huge negative number. So it's a better representation now

**Review by fabiooshiro — APPROVED** [review]
LGTM!

**Review by sandhilt — APPROVED** [review]
LGTM 👍🏽

**Review by sandhilt — APPROVED** [review]
LGTM 👍🏽

**Pull request merged**